sycl/test-e2e/format.py

```diff
@@ -203,9 +203,9 @@ def get_extra_env(sycl_devices):
                 # Expand device-specific condtions (%if ... %{ ... %}).
                 tmp_script = [ cmd ]
                 conditions = {x: True for x in sycl_device.split(':')}
-                for op_sys in ['linux', 'windows']:
-                    if op_sys in test.config.available_features:
-                        conditions[op_sys] = True
+                for cond_features in ['linux', 'windows', 'preview-breaking-changes-supported']:
+                    if cond_features in test.config.available_features:
+                        conditions[cond_features] = True
```

Comment:
@aelovikov-intel - Would it make sense to maybe just set all available features as true conditions?

Reply:
We have those `gpu-intel-*` and `matrix-*` that won't be accessible to all devices. I think technically the only legal run is when these options are specified together with proper `sycl_devices` parameter. https://github.com/intel/llvm/blob/106bc89bacc7800de52912b8a35d74373e33fcc0/sycl/test-e2e/lit.cfg.py#L307-L315 might also be problematic.